### PR TITLE
New theme and first field notes

### PR DESCRIPTION
**Pull request by pelson**
In this edition, I talk about Heroku, Docker and Dropbox.

### CODE REVIEW

File: content/heroku_fuse_mount.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    2017-02-07T14:36:06.997273+00:00 app[web.1]: RuntimeError: 1
+
+In addition, there is a message in the log along the lines of:
+
+    fuse: device not found, try 'modprobe fuse' first
+
+It may be as I feared: Heroku doesn't currently support FUSE mounts.
+
+In order to get one final datapoint, I put together the equivalent Dockerfile for ubuntu rather than alpine. Unfortunately the results are the same.
+
```
Comment (pelson) [inline content/heroku_fuse_mount.md]: Would be great to have a twitter feed in here relating.

File: content/heroku_fuse_mount.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    brew install osxfuse
+
+Again, no luck, but homebrew does point me in the direction of a cask version (pre-compiled, rather than building it from source on my machine). I'm pretty keen to get going with this, so I go ahead and install the casked version:
+
+    brew install Caskroom/cask/osxfuse
+
+After a short wait, the result is positive, and it looks like I have a local FUSE installation.
+
+Next, I want to try it out. Rather than choose to do somethign simple, I go straight for the jugular and try to get a dropbox FUSE mount working. I clone https://github.com/realriot/ff4d and get myself set up with a legacy python installation (in a clean environment named "ff4d_py2":
```
Comment (ocefpaf) [inline content/heroku_fuse_mount.md]: somethign -> something :stuck_out_tongue_closed_eyes:

File: content/heroku_fuse_mount.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+It is this address (+":5000") that I use to see the webapp in my browser.
+
+Things are starting to come together nicely, except the mount of my device was failing with messages similar to:
+
+    Starting FUSE...
+    fuse: failed to open /dev/fuse: Operation not permitted
+    [ERROR] Failed to start FUSE... (Traceback (most recent call last):
+
+It turns out that we need elevated priveledges to mount a FUSE device within docker.
+Adding ```--cap-add SYS_ADMIN``` and ```--device /dev/fuse``` to docker should be enough (though it appears there was once a bug in docker that meant the container needed to be run with *full* priveledges).
```
Comment (ocefpaf) [inline content/heroku_fuse_mount.md]: priveledges -> privileges :stuck_out_tongue_winking_eye:

Comment (pelson) [inline content/heroku_fuse_mount.md]: Thanks @ocefpaf. Maybe a spell-checker would come in handy! :smile:

**Pull request merged**